Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2436,6 +2436,23 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
 // TODO: Consider making the return type be generic over `FloatingPoint` types
 // so that `self`'s scalar type can be any `Numeric` type.
 public extension Tensor where Scalar: TensorFlowFloatingPoint {
+
+    /// Helper function that assess if `axis` is in the range `[-rank, rank)`, where `rank` is the rank of
```

Comment:
The current documentation comment is not accurate: `isAxisInRange` does not perform an assertion (which contradicts `Helper function that assess if ...`).

Suggestion:
```swift
    /// Returns `true` if the given axis is in the range `[-rank, rank)`.
```